python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1345,3 +1345,40 @@ def test_groupby_apply_return_series_dataframe(cust_func):
     actual = gdf.groupby(["key"]).apply(cust_func)
 
     assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize(
+    "pdf", [pd.DataFrame(), pd.DataFrame({"a": []}), pd.Series([])]
+)
+def test_groupby_no_keys(pdf):
+    gdf = cudf.from_pandas(pdf)
+    assert_eq(
+        pdf.groupby([]).max(),
+        gdf.groupby([]).max(),
+        check_dtype=False,
+        check_index_type=False,
```

Comment:
How do we differ here?

Reply:
We produce a `Float64Index` while Pandas produces an `Int64Index`. Not sure exactly why

Reply:
Can we add a comment with this in the relevant area in `groupby.py` just in case it comes up as an issue we have a reference?

Reply:
Done